thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -369,7 +436,16 @@ export default Route.extend(AuthenticatedRouteMixin, {
       selectedUrns,
       sizeMetricUrns,
       setupMode,
-      context
+      context,
+
+      // reset overrides
+      contextUrnsInit: undefined,
```

Comment:
Could this be reset in resetController?

Reply:
I'm not happy with this thing overall. It should wipe any query params off the RCA url once the RCA is set up